documentation/modules/deploying/proc-deploy-topic-operator-standalone.adoc

```diff
@@ -175,3 +175,49 @@ strimzi-topic-operator  1/1    1           1
 +
 `READY` shows the number of replicas that are ready/expected.
 The deployment is successful when the `AVAILABLE` output shows `1`.
+
+== (Preview) Deploying the standalone Topic Operator for unidirectional topic management
+
+Unidirectional topic management maintains topics solely through `KafkaTopic` resources.
+For more information on unidirectional topic management, see xref:ref-operator-topic-{context}[].
+
+If you want to try the preview of unidirectional topic management, follow these steps to deploy the standalone Topic Operator:
+
+. Undeploy the current standalone Topic Operator.
++
+Retain the `KafkaTopic` resources, which are picked up by the Topic Operator when it is deployed again.
+
+. Edit the `Deployment` configuration for the standalone Topic Operator to remove any ZooKeeper-related environment variables:
++
+* `STRIMZI_ZOOKEEPER_CONNECT`
+* `STRIMZI_ZOOKEEPER_SESSION_TIMEOUT_MS`
+* `TC_ZK_CONNECTION_TIMEOUT_MS`
+* `STRIMZI_USE_ZOOKEEPER_TOPIC_STORE`
++
+Unidirectional topic management does not use ZooKeeper.
++
+Other unused environment variables that can be removed if present:
++
+* `STRIMZI_REASSIGN_THROTTLE`
+* `STRIMZI_REASSIGN_VERIFY_INTERVAL_MS`
+* `STRIMZI_TOPIC_METADATA_MAX_ATTEMPTS`
+* `STRIMZI_TOPICS_PATH`
+* `STRIMZI_STORE_TOPIC`
+* `STRIMZI_STORE_NAME`
+* `STRIMZI_APPLICATION_ID`
+* `STRIMZI_STALE_RESULT_TIMEOUT_MS`
```

Comment:
Why the 4 env vars above _should be removed_ and these only _can be removed_? Not sure I follow what makes them different.

Reply:
I don't think UTO works if the ZK variables are present. I assumed the other variables would be ignored if present.   I'll see what Tom says on this split

Reply:
I think we should clarify it in some dedicated paragraph. Because it is not completely clear that it is the presence or absence of the ZK variables which defines which operator is used. The Feature Gates as Tom uses it do not propagate into the Topic Operator. So for standalone TO users this is important information.

Reply:
For the standalone TO it is the absence of the ZK env vars that determines which operator will be used. For the entity topic operator the +UnidirectionalTopicOperator feature gate determines which env vars it will be deployed with.

Reply:
Updated with:

> It is the presence or absence of the ZooKeeper variables that defines whether the unidirectional Topic Operator is used.
> Unidirectional topic management does not use ZooKeeper.
> If ZooKeeper environment variables are not present, the unidirectional Topic Operator is used.
> Otherwise, the bidirectional Topic Operator is used.